account_invoice_merge_purchase/README.rst

```diff
@@ -1,26 +1,33 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+==================================
 Merge invoices from purchase order
 ==================================
 
-This module provides a compatibility between purchase order workflow and account invoice merge module
+This module provides a compatibility between purchase order workflow and
+account invoice merge module
 
 
 Usage
 =====
 
+Select several vendor bills from the same partner and click on **Merge Partner
+Invoice** in Action button menu.
+
```

Comment:
Hi, I don't see this option:
![2017-03-07_18-01-08](https://cloud.githubusercontent.com/assets/8736623/23668041/1bfbdcf6-0360-11e7-9a76-df8e62f9cdb5.png)